### PR TITLE
[TEST-293] - Unable to run MicroProfile TCK Health on payara-micro-managed

### PR DESCRIPTION
**Pull request by cubastanley**
Caused by unwanted dependency inheritance in tck-arquillian-extension pom
Tested against micro-managed, server-managed and server-remote

### CODE REVIEW

File: MicroProfile-Health/tck-runner/pom.xml [hunk 1]
```diff
@@ -69,18 +69,29 @@
...
             <scope>test</scope>
         </dependency>
+        
+        <!-- CAUSES ISSUES WITH MICRO MANAGED -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>tck-arquillian-extension</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+            <exclusions>
```
Comment (MattGill98) [inline MicroProfile-Health/tck-runner/pom.xml]: Why does the artifact include this in the first place? That seems like the root of the problem

Comment (cubastanley) [reply MicroProfile-Health/tck-runner/pom.xml]: You're not wrong, and honestly I can't tell you - dependency isn't stated anywhere in the parent pom and NetBeans won't remove it so this was the best way I could think of to resolve the issue

Comment (MattGill98) [reply MicroProfile-Health/tck-runner/pom.xml]: As a hint, it'll be an inherited dependency. I would suggest looking further up the tree and potentially moving the dependency further down if it's possible

Comment (cubastanley) [reply MicroProfile-Health/tck-runner/pom.xml]: That's what I mean, the top level pom (TCK-parent) doesn't have a generally available dependency on that container, from having looked at it, it appears to be coming in with an external dependency group that's also importing required test dependencies

Comment (cubastanley) [reply MicroProfile-Health/tck-runner/pom.xml]: I'll have another look and confirm

**Pull request merged**